src/prices/constants/prices.constants.ts

```diff
@@ -0,0 +1,7 @@
+export const PRICES_API: { [key: string]: any } = {
+  PRICES_DEFAULT: 'https://api.raporty.pse.pl/api/',
```

Comment:
I think it's usually better if base url doesn't have a trailing `/` at the end.

For example later you construct an url like this:

```typescript
const url: string = `${PRICES_API.PRICES_DEFAULT}${PRICES_API.REPORTS.CURRENT_PRICES(convertedDate)}`;
```

without trailing `/` we would construct it like this:

```typescript
const url: string = `${PRICES_API.PRICES_DEFAULT}/${PRICES_API.REPORTS.CURRENT_PRICES(convertedDate)}`;
```